src/components/CurrentUserPersonalDetailsSkeletonView/index.tsx

```diff
@@ -1,38 +1,35 @@
-import PropTypes from 'prop-types';
 import React from 'react';
+import SkeletonViewContentLoader from 'react-content-loader/native';
```

Comment:
This component loader only works on the native platform.

```suggestion
import SkeletonViewContentLoader from 'react-content-loader/native';
import SkeletonViewContentLoader from '@components/SkeletonViewContentLoader';
```

Reply:
Updated!